### PR TITLE
Ref 716

### PR DESCRIPTION
**Pull request by brettminnie**
* added support for asg scaling policies

### CODE REVIEW

File: roles/ec2-app/node-metadata.md [hunk 1]
```diff
@@ -58,6 +58,34 @@ Examples
...
+      scaling_policies:
+        - name: scale_up
+          adjustment_type: "Percent"
+          min_adjustment_step: +1
+          scaling_adjustment: 100
+          cooldown: 180
+          metric_alarms:
+            - metric: "CPUUtilization"
+              comparison: ">="
+              threshold: 40
```
Comment (nsmithuk) [inline roles/ec2-app/node-metadata.md:70]: @brettminnie not 45 here?

Comment (nsmithuk) [reply roles/ec2-app/node-metadata.md:70]: oh - it get overridden in the local env pillar?

Comment (brettminnie) [reply roles/ec2-app/node-metadata.md:70]: That's example data, part of why I said don't merge, need to add the dynamodb stuff there too

File: roles/ec2-dynamodb/tasks/create-scaling-policies.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            --service-namespace dynamodb
+            --resource-id 'table/{{ dynamodb_scaling_policy.name }}-{{ target }}'
+            --scalable-dimension 'dynamodb:table:WriteCapacityUnits'
+             --min-capacity {{ dynamodb_scaling_policy.write_min_units }}
+            --max-capacity {{ dynamodb_scaling_policy.write_max_units }}
+            --role-arn {{ role_arn }}"
+  register: dynamodb_scalable_write_target
+
+- name: Create our temporary write scaling policy file
+  template:
```
Comment (carinadigital) [inline roles/ec2-dynamodb/tasks/create-scaling-policies.yml:35]: Minor, but could you template to a variable instead of a file.

**Pull request merged**